src/App/Utilities/constants.js

```diff
@@ -0,0 +1,288 @@
+        </>
+    }
+];
+
+export const displayHidingElements = [
+    {
+        id: "displaynotation",
+        title: "Notation",
+        subtitle: "Quick display classes. d(isplay)-<value>",
+        content: <>
+            {hidingElementClasses.notation.map((rule, i) => (
+                <DisplayPresenter key={rule + i} util={rule.value} css={rule.css}/>
+            ))}
+        </>
+    },
+    {
+        id: "hidingElmenents",
+        title: "Hiding elements",
+        subtitle: "For faster mobile-friendly development",
+        content: <>
+            {hidingElementClasses.hidingElements.map((rule, i) => (
+                <DisplayPresenter key={rule + i} util={rule.screenSize} css={rule.dClass} textVersion/>
+            ))}
+        </>
+    }
+];
+
+export const flex = [
+    {
+        id: "enableflexbahaviour",
+        title: "Enable flex bahaviour",
+        subtitle: "For creating a flexbox",
+        content: <>
+            {flexClasses.flexBehaviour.map((rule, i) => (
+                <FlexPresenter key={rule + i} code={rule.code}/>
+            ))}
+        </>
+    },
+    {
+        id: "flexDirection",
+        title: "Direction",
+        subtitle: "Direct the flex",
+        content: <>
+            {flexClasses.direction.map((rule, i) => (
+                <FlexPresenter key={rule + i} code={rule.code} />
+            ))}
+        </>
+    },
+    {
+        id: "justifyContent",
+        title: "Justify content",
+        subtitle: "Justify your flex contents",
+        content: <>
+            {flexClasses.justifyContent.map((rule, i) => (
+                <FlexPresenter key={rule + i} code={rule.code} />
+            ))}
+        </>
+    },
+    {
+        id: "alignItems",
+        title: "Align items",
+        subtitle: "Align set of elements",
+        content: <>
+            {flexClasses.alignItems.map((rule, i) => (
+                <FlexPresenter key={rule + i} code={rule.code} />
+            ))}
+        </>
+    },
+    {
+        id: "alignSelf",
+        title: "Align self",
+        subtitle: "Align element",
+        content: <>
+            {flexClasses.alignSelf.map((rule, i) => (
+                <FlexPresenter key={rule + i} code={rule.code} />
+            ))}
+        </>
+    },
+    {
+        id: "fill",
+        title: "Fill",
+        subtitle: "Flex fill",
+        content: <>
+            {flexClasses.fill.map((rule, i) => (
+                <FlexPresenter key={rule + i} code={rule.code} />
+            ))}
+        </>
+    },
+    {
+        id: "growAndShrink",
+        title: "Grow and shrink",
+        subtitle: "Effective use of space",
+        content: <>
+            {flexClasses.growAndShrink.map((rule, i) => (
+                <FlexPresenter key={rule + i} code={rule.code} />
+            ))}
+        </>
+    },
+    {
+        id: "wrap",
+        title: "Wrap",
+        subtitle: "Nowrap is a browsers defualt",
```

Comment:
typo (+ if touching it then small rephrasing)
```suggestion
        subtitle: "Nowrap is the browser's default value",
```